lib/cfn-model/model/cfn_model.rb

```diff
@@ -60,6 +60,10 @@ def standalone_egress
     end
   end
 
+  def resources_by_id(resource_id)
```

Comment:
should this be singular (and thus a find instead of select?)

Reply:
Yep, good catch!  Updated in latest commit.